dsp/modules/google.py

```diff
@@ -101,4 +148,52 @@ def __call__(
         return_sorted: bool = False,
         **kwargs
     ):
-        return self.request(prompt, **kwargs)
+        assert only_completed, "for now"
+        assert return_sorted is False, "for now"
+
+        n = kwargs.pop("n", 1)
+
+        completions = []
+        for i in range(n):
+            response = self.request(prompt, **kwargs)
+            completions.append(response.parts[0].text)
+
+        return completions
+
+    def inspect_history(self, n: int = 1, skip: int = 0):
```

Comment:
Do we need this function instead of the existing one in `lm.py` file?

Reply:
I can see that in `lm.py` the response is handling response based on the `provider`. For now DSPy supports few models but in my opinion this is not scalable when new models are added since each model has its own response format. Modifying `lm.py` to support Google response schema would work fine too, so it can be another options.